build/automation/lib/docker.mk

```diff
@@ -9,16 +9,16 @@ DOCKER_NETWORK = $(PROJECT_GROUP_SHORT)/$(PROJECT_NAME_SHORT)/$(BUILD_ID)
 DOCKER_REGISTRY = $(AWS_ECR)/$(PROJECT_GROUP_SHORT)/$(PROJECT_NAME_SHORT)
 DOCKER_LIBRARY_REGISTRY = nhsd
 
-DOCKER_ALPINE_VERSION = 3.14.2
+DOCKER_ALPINE_VERSION = 3.15.0
 DOCKER_COMPOSER_VERSION = 2.0.13
 DOCKER_CONFIG_LINT_VERSION = v1.6.0
-DOCKER_DIND_VERSION = 20.10.8-dind
-DOCKER_EDITORCONFIG_CHECKER_VERSION = 2.3.5
-DOCKER_ELASTICSEARCH_VERSION = 7.13.0
+DOCKER_DIND_VERSION = 20.10.12-dind
+DOCKER_EDITORCONFIG_CHECKER_VERSION = 2.4.0
+DOCKER_ELASTICSEARCH_VERSION = 7.17.0
```

Comment:
I think when this was upgraded before it cause SF problems with the create snapshot make target.  But I'm not worried about it as we (SF) can fix that easily enough when they next refresh the devops scripts.

Reply:
Thanks for that Jon. Probably, in that case, the best thing for the SF is to keep the version that is known to work.